src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -0,0 +1,899 @@
+                            error={
+                                shouldDisplayMerchantError || (shouldDisplayFieldError && TransactionUtils.isMerchantMissing(transaction ?? null))
+                                    ? translate('common.error.enterMerchant')
+                                    : ''
+                            }
+                        />
+                    )}
+                    {shouldShowCategories && (
+                        <MenuItemWithTopDescription
+                            shouldShowRightIcon={!isReadOnly}
+                            title={iouCategory}
+                            description={translate('common.category')}
+                            numberOfLinesTitle={2}
+                            onPress={() => {
+                                if (isEditingSplitBill) {
+                                    Navigation.navigate(ROUTES.EDIT_SPLIT_BILL.getRoute(reportID ?? '', reportActionID ?? '', CONST.EDIT_REQUEST_FIELD.CATEGORY));
+                                    return;
+                                }
+                                Navigation.navigate(ROUTES.MONEY_REQUEST_CATEGORY.getRoute(iouType, reportID));
+                            }}
+                            style={styles.moneyRequestMenuItem}
+                            titleStyle={styles.flex1}
+                            disabled={didConfirm}
+                            interactive={!isReadOnly}
+                            rightLabel={canUseViolations && Boolean(policy?.requiresCategory) ? translate('common.required') : ''}
+                        />
+                    )}
+                    {shouldShowTags && (
+                        <MenuItemWithTopDescription
+                            shouldShowRightIcon={!isReadOnly}
+                            title={PolicyUtils.getCleanedTagName(iouTag)}
+                            description={policyTagListName}
+                            numberOfLinesTitle={2}
+                            onPress={() => {
+                                if (isEditingSplitBill) {
+                                    Navigation.navigate(
+                                        ROUTES.MONEY_REQUEST_STEP_TAG.getRoute(
+                                            CONST.IOU.ACTION.EDIT,
+                                            CONST.IOU.TYPE.SPLIT,
+                                            transaction?.transactionID ?? '',
+                                            reportID ?? '',
+                                            Navigation.getActiveRouteWithoutParams(),
+                                        ),
+                                    );
+                                    return;
+                                }
+
+                                Navigation.navigate(ROUTES.MONEY_REQUEST_TAG.getRoute(iouType, reportID));
```

Comment:
Hey @mountiny @roryabraham @hoangzinh  there is no route `MONEY_REQUEST_TAG` defined so not sure if this line is a mistake or what

Reply:
@hoangzinh thanks will update that today!

Reply:
@hoangzinh hey, when I am adding that I am landing on Not found page, are you sure this is correct?

Reply:
I just hardcoded `shouldShowTags` to true , but I see what is the problem there is no `transaction` so we are not providing `transactionID` to url. So in case if we are not in editing split bill we will never have this `transactionID` so is this even necessary?

Reply:
@hoangzinh if you have time please recheck 😄